dataland-backend/src/main/kotlin/org/dataland/datalandbackend/health/HealthIndicatorExtensionForDummyServers.kt

```diff
@@ -0,0 +1,45 @@
+package org.dataland.datalandbackend.health
+
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import org.springframework.boot.actuate.health.Health
+import org.springframework.boot.actuate.health.HealthIndicator
+
+/**
+ * This class customizes the actuator/health endpoint of the backend to display if the given server
+ * is available.
+ */
+open class HealthIndicatorExtensionForDummyServers(private val basePath: String) : HealthIndicator {
```

Comment:
Ich denke das Backend sollte nichts enthalten, was wir nur für's testen brauchen!

Reply:
@SiegfriedSobkowiak  du erinnerst dich bestimmt:
Als wir im Proxy direkte Routes zu den Health Endpoints der Dummy Server umsetzen wollten, hatten wir ständig Probleme.

Reply:
Vllt noch ein Argument dafür?:

Wir wollen in den Tests, die diesen Health-Endpunkt hier nutzen, ja auch eigentlich wissen, ob das **Backend** sagt, dass es diese Dummy-Server erreichen kann, und nicht ob die Test-App die Dummy-Server erreichen kann.

Somit ist die Aussagekraft der Health-Tests ja eigentlich besser, wenn sie testen, ob das Backend die Dummy-Server erreicht.

Reply:
ich setzte das um wie besprochen

Reply:
erledigt

Reply:
ok